src/Uno.UI.Composition/Composition/Visual.skia.cs

```diff
@@ -178,7 +195,17 @@ internal virtual void Render(in DrawingSession parentSession)
 		}
 
 		using var session = BeginDrawing(in parentSession);
-		Draw(in session);
+		if (_requiresRepaint)
+		{
+			_requiresRepaint = false;
+			_recorder ??= new SKPictureRecorder();
```

Comment:
Since we are actually interested only by the `_picture`, can we pool the `SKPictureRecorder` instead of having one per `Visual`?